types/react-native/index.d.ts

```diff
@@ -11,6 +11,7 @@
 //                 Michele Bombardi <https://github.com/bm-software>
 //                 Tanguy Krotoff <https://github.com/tkrotoff>
 //                 Alexander T. <https://github.com/a-tarasyuk>
+//                 Alex Rodin <https://github.com/nidorx>
```

Comment:
Are you aware that this is not just a list of people that have worked on the types, but means that you are signing up to be a maintainer and will get asked to review PRs and issues in the future?

Reply:
@andy-ms Do you think it might be helpful for the bot to point this out? I’ve noticed more than once that people don’t understand what this list is really used for.

Reply:
2nd that!